specification/security/resource-manager/Microsoft.Security/stable/2019-08-01/iotSecuritySolutions.json

```diff
@@ -636,8 +639,51 @@
         "$ref": "#/definitions/RecommendationConfigurationProperties"
       }
     },
+    "AdditionalWorkspaceProperties": {
+    "type": "object",
+    "description": "The properties of an additional workspace",
+    "properties": {
+      "workspace": {
+        "type": "string",
+        "description": "The ARM ID of the workspace"
+      },
+      "region": {
+        "type": "string",
+        "description": "The region that the workspace resides in"
+      },
+      "type": {
+        "type": "string",
+        "description": "The type of the workspace",
+        "enum": [
+          "Sentinel"
```

Comment:
If you only have one enum value for type, does it need to be in the model?

Reply:
Currently the only option is Sentinel, but we wanted to future-proof this.